vm/package/deploy.rs

```diff
@@ -192,14 +200,14 @@ mod tests {
         let (directory, package) = crate::package::test_helpers::sample_package();
 
         // Deploy the package.
-        let deployment = package.deploy::<CurrentAleo>(None).unwrap();
```

Comment:
shouldn't these tests be updated to include the transaction id field you added?

Reply:
Since the test cases use `None` for the endpoint (because there isn't an actual deployment) the `transaction_id` will always be an empty string because there is not an actual transaction. Do you think is worth adding it to the test anyway?

Reply:
Can the test be rewritten to send a Some endpoint and yield a non empty transaction? Otherwise it seems we're only testing the getters and setters

Reply:
That's true, is not that useful to have those tests here. Maybe we have to consider doing them in another repo where we can test a real transaction with a real endpoint.